index.html

```diff
@@ -40,252 +46,272 @@
+
+  <p>The Working Group intends to develop the following Technical Reports:</p>
+
+  <dl>
+    <dt class="spec">
+      <a href="https://www.w3.org/TR/ttml3/">Timed Text Markup Language 3 (TTML3)</a>
+    </dt>
+
+    <dd>
+      <p>This specification defines a content type that represents timed text media for the purpose of
+      interchange among authoring, distribution and playback systems. Timed text is textual information that is
+      intrinsically or extrinsically associated with timing information.</p>
+    </dd>
+
+    <dt id="webvtt" class="spec">
+      <a href="https://www.w3.org/TR/webvtt1/">WebVTT: The Web Video Text Tracks Format</a>
+    </dt>
+
+    <dd>
+      <p>This specification defines WebVTT, the Web Video Text Tracks format. Its main use is for marking up
+      external text track resources in connection with the HTML &lt;track&gt; element. WebVTT files provide
+      captions or subtitles for video content, and also text video descriptions <a href=
+      "http://www.w3.org/TR/media-accessibility-reqs/">[MAUR]</a>, chapters for content navigation,
+      and more generally any form of metadata that is time-aligned with audio or video content.</p>
+    </dd>
+
+    <dt id="adpt" class="spec">TTML Profile for Audio Description</dt>
+
+    <dd>
+      <p>This specification defines a profile of [ <cite><a class="bibref" href=
+      "https://www.w3.org/TR/ttml2/">TTML2</a></cite> ] intended to support audio description script exchange
+      throughout the workflow including production of the script, rendering as voice by recording or text to
+      speech synthesis, and audio mixing.</p>
+    </dd>
+  </dl>
+
+  <p>The Working Group MAY develop additional Recommendation-track and non-Recommendation-track Technical
+  Reports.</p>
+</div>
+
+<div id="revisions">
+  <h3>Existing Technical Reports</h3>
+
+  <p>The Working Group MAY update its <a href="https://www.w3.org/wiki/TimedText/Publications">previously
+  published Technical Reports</a>.</p>
+</div>
+<!-- END here: copy from 2019 charter -->
+
+<div id="ig-other-deliverables">
+  <h3>Other deliverables</h3>
+
+  <p>The Working Group MAY create documents that are not Technical Reports, including:</p>
+
+  <ul>
+    <li>Use case and requirement documents</li>
+
+    <li>Test suite and implementation reports</li>
+
+    <li>Primer or Best Practice documents to support web developers</li>
+  </ul>
+</div>
+
+        <section id="timeline">
+          <h3>Timeline</h3>
+            <p class="todo">Put here a timeline view of all deliverables.</p>
+            <ul class="todo">
+              <li>Month YYYY: First teleconference</li>
+              <li>Month YYYY: First face-to-face meeting</li>
+              <li>Month YYYY: Requirements and Use Cases for FooML</li>
+              <li>Month YYYY: FPWD for FooML</li>
+              <li>Month YYYY: Requirements and Use Cases for BarML</li>
+              <li>Month YYYY: FPWD FooML Primer</li>
+            </ul>
```

Comment:
how about to write something like : https://www.w3.org/2021/06/web-editing-wg-charter.html#timeline

Reply:
Something like that, yes - I think it would be good to say "current timelines are documented at [link] and will be updated on an occasional basis" too, just so there's some pointer.

Reply:
replaced.